<launch>
  <node
    pkg="rqt_reconfigure"
    type="rqt_reconfigure"
    name="rqt_reconfigure" 
    output="screen"
  />
  <node
    pkg="rviz"
    type="rviz"
    name="rviz"
    args="-d $(find tofcore_ros1)/rviz/default_config.rviz"
    output="screen"
  />
  <node
    pkg="tofcore_ros1"
    type="tof_sensor"
    name="tof_sensor"
    output="screen"
  />
  </launch> 